opm/autodiff/FlowMain.hpp

```diff
@@ -427,7 +429,7 @@ namespace Opm
             OpmLog::addBackend( "STREAMLOG", streamLog);
             std::shared_ptr<StreamLog> debugLog = std::make_shared<EclipsePRTLog>(debugFile, Log::DefaultMessageTypes, false, output_cout_);
             OpmLog::addBackend( "DEBUGLOG" ,  debugLog);
-            const auto& msgLimits = eclipse_state_->getSchedule().getMessageLimits();
+            const auto& msgLimits = schedule_->getMessageLimits();
```

Comment:
Before this is called, we must create the schedule object and the summary config object (just null pointers here). I suggest doing so just after line 505. (Sorry for confusing method order, but earlier the logging was set up before the deck was read so that is why the readDeckInput() method is after setupLogging() in the file.)

The following lines were applied there successfully:
```
                schedule_.reset(new Schedule(*deck_,
                                             eclipse_state_->getInputGrid(),
                                             eclipse_state_->get3DProperties(),
                                             eclipse_state_->runspec().phases(),
                                             parseContext));
                summary_config_.reset(new SummaryConfig(*deck_,
                                                        *schedule_,
                                                        eclipse_state_->getTableManager(),
                                                        parseContext));
```

With this, all tests succeed for me.

Reply:
Thank you for debug assistance.